### PR TITLE
Fix typo in deleteExternalResources comment

### PR DESCRIPTION
**Pull request by n1r1**
I believe the intention was to explain idempotence, so this is just a small typo correction

### CODE REVIEW

**Comment by camilamacedo86** [comment]
Hi @n1r1, 

Tks for your collab. See that the CI is not passing. Mainly you need to assign in the CLA and run make generated to update the testdata with. See the CONTRIBUTION.md doc for a better understanding.

**Comment by n1r1** [comment]
Thanks @camilamacedo86 
I've signed the CLA and added the files after `make generate`

/retest

**Comment by n1r1** [comment]
I signed it

**Comment by camilamacedo86** [comment]
HI @n1r1, 

You need to rebase this on with the master, squash the commits, and ensure that you have the SLA because if it does not pass in the CI is not possible to merge. Could you please check?

File: testdata/project-v2-addon/go.mod [hunk 1]
```diff
@@ -9,5 +9,5 @@ require (
 	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v0.17.2
 	sigs.k8s.io/controller-runtime v0.5.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200415210853-85eb326a6add
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200505212709-bc9d91267442
```
Comment (camilamacedo86) [inline testdata/project-v2-addon/go.mod:12]: this should not be part of the pr. Could you please rebase with the master?

**Comment by n1r1** [comment]
@camilamacedo86 , I've signed the CLA and rebased with master.
`make generate` didn't yield any changes this time, but `make check-testdata` passed on my end. I hope this is okay.

Let me know if anything else is needed.
thanks
